app/federation/server/main.js

```diff
@@ -22,6 +22,8 @@ export const Federation = {
 	uniqueId: null,
 	localIdentifier: null,
 };
+export const PeerClient = peerClient;
```

Comment:
instead of exporting `PeerClient` here (which is a reference of `peerClient`) I would recommend to import `peerClient` directly where ever is it needed.

Reply:
you realized that this is going to be initialized later on the `main.js` code, right? I will still have to export this specific resource somewhere.

Reply:
which is still fine, since it's modifying the original object.

I just want to be consistent, there are some places already importing `peerClient` (like [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/federation/server/federatedResources/FederatedMessage.js#L9))

Reply:
oh, you mean the name? I thought it was "load directly from the file". Ok, got it.